### DISCUSSION TITLE
主动退群怎么实现

### DISCUSSION THREAD

**Discussion by ityaquan** [simple-robot/Discussions#99] [Q&A] [answered]
### 涉及的编程语言

Java

### 疑问描述

MiraiBotJoinGroupEvent这个类下貌似没有退群方法

**Comment by ForteScarlet** [comment, marked as answer]
```java
MiraiGroup group = ...;
group.deleteBlocking(); // 退群
```

https://docs.simbot.forte.love/components/mirai/simbot-component-mirai-core/love.forte.simbot.component.mirai/-mirai-group/index.html

**Reply by ityaquan** [reply]
有效，感谢
